src/shared_modules/indexer_connector/src/monitoring.hpp

```diff
@@ -146,15 +150,32 @@ class Monitoring final
                                         }
                                         else
                                         {
+                                            logWarn(IC_MONITORING,
```

Comment:
Question: is it Ok to print this failure as a WARNING message but the other ones as DEBUG?
We should unify the criteria.

If we consider that the requests are run every 60 seconds, I vote to use DEBUG for all of them-

Reply:
done